src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -4,14 +4,30 @@
 
 package frc.robot.subsystems;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class Intake extends SubsystemBase {
   /** Creates a new Intake. */
-  public Intake() {}
+
+   CANSparkMax motorController1; 
+   CANSparkMax motorController2; 
+
+  public Intake() {
+    motorController1 = new CANSparkMax(0, MotorType.kBrushless);
```

Comment:
Remember to change the IDs of the motor controllers since they're being used by the drivetrain.

Reply:
the id should also be a constant

Reply:
it might also be better to rename the variables to something like topSpark and bottomSpark so that its more clear what it corresponds to on the robot